eng/scripts/Language-Settings.ps1

```diff
@@ -301,6 +313,15 @@ function UpdateDocsMsPackages($DocConfigFile, $Mode, $DocsMetadata) {
         -Value $packageVersion `
         -PassThru `
         -Force 
+      if ($PackageSourceOverride) {
```

Comment:
This logic should also happen near line 364 below to cover cases where packages have only ever published dev versions to the package feed and never to PyPI

Reply:
After a second thoughts, the alpha packages only get updated through pipeline. If a package does not showing up in json files but in csv metadata, then we do not have a chance to update the package to alpha version. So we currently do not have a case in `remaining packages`